deploy/docs/ContainerLogs.md

```diff
@@ -0,0 +1,220 @@
+# Container log parsing
```

Comment:
This new documentation article is only linked in a comment in Fluent Bit configuration. Shouldn't we also link to it from some other doc (`.md` file)?

Reply:
I added link to this document in [values.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1441/files#diff-da997ab241c9344f2bb085822fe04e4e43eaf0f8034a9d15dc3942b8a5fbe046R850)